test/browser/component.js

```diff
@@ -126,6 +126,25 @@ describe(`Simple Component instance`, function() {
     });
   });
 
+  context(`when detached from DOM`, function() {
+    beforeEach(async function() {
+      document.body.appendChild(el);
+      await nextAnimationFrame();
+      document.body.removeChild(el);
+      await nextAnimationFrame();
+    });
+
+    it(`disconnectedCallback cleans up to be GC friendly`, function() {
```

Comment:
spec docstring format = `"it <does something>"`
```suggestion
    it(`cleans up references to be GC friendly`, function() {
```